gem/client/css/state/_s-article-preview.scss

```diff
@@ -142,9 +142,9 @@
     width: 100%;
     display: block;
     content: " ";
-    //background: url(/static/img/pattern_edited.png);
+    background: url(/static/img/featured_bar_rwanda.png);
     margin: 0 0 21px 0;
-    background-size: 100% auto;
+    /*background-size: 100% auto;*/
```

Comment:
Great, I will remove that now.
The whole site needs to be refactored in the next sprint.